contrib/opencensus-ext-azure/README.rst

```diff
@@ -114,6 +69,109 @@ You can enrich the logs with trace IDs and span IDs by using the `logging integr
         logger.warning('In the span')
     logger.warning('After the span')
 
+Metrics
+~~~~~~~
+
+The **OpenCensus Azure Monitor Metrics Exporter** allows you to export metrics to `Azure Monitor`_.
+
+* Create an Azure Monitor resource and get the instrumentation key, more information can be found `here <https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource>`_.
+* Put the instrumentation key in ``APPINSIGHTS_INSTRUMENTATIONKEY`` environment variable.
+* You can also specify the instrumentation key explicitly in the code, which will take priority over a set environment variable.
+
+    .. code:: python
+
+        import time
+
+        from opencensus.ext.azure import metrics_exporter
+        from opencensus.stats import aggregation as aggregation_module
+        from opencensus.stats import measure as measure_module
+        from opencensus.stats import stats as stats_module
+        from opencensus.stats import view as view_module
+        from opencensus.tags import tag_map as tag_map_module
+
+        stats = stats_module.stats
+        view_manager = stats.view_manager
+        stats_recorder = stats.stats_recorder
+
+        CARROTS_MEASURE = measure_module.MeasureInt("carrots",
+                                                    "number of carrots",
+                                                    "carrots")
+        CARROTS_VIEW = view_module.View("carrots_view",
+                                        "number of carrots",
+                                        [],
+                                        CARROTS_MEASURE,
+                                        aggregation_module.CountAggregation())
+
+
+        def main():
+            # Enable metrics
+            # Set the interval in seconds in which you want to send metrics
+            exporter = metrics_exporter.new_metrics_exporter(export_interval=2)
```

Comment:
Minor suggestion, it is not clear if the interval should be an integer or it can be a float/decimal, consider either update the comment or put something like `2.0` (if we do support float).

Reply:
We allow floats so I will put 2.0 in the sample to let users know implicitly this is possible.

Reply:
Great, this is something that we've been practicing, e.g. `ProbabilitySampler(1.0)` and
```
aggregation_module.DistributionAggregation(
        [100.0, 200.0, 400.0, 1000.0, 2000.0, 4000.0]))
```